setup/4.install_ganache.sh

```diff
@@ -9,7 +9,7 @@ export KONG_PROXY_PORT=$(kubectl get service gateway-kong-proxy -o json | jq -r
 curl -k -X POST \
   --url $KONG_ADMIN_URL/apis/ \
   --data 'name=ganache' \
-  --data 'uris=/ganache' \
+  --data 'hosts=ganache.transmute.minikube' \
```

Comment:
This won't work if they haven't already configured their hosts file, right?

Reply:
it will work, but they will have to curl with a hosts header. 

This is difference should have been resolved a while ago.